File: client/public/Codes/Java/OddOrEven.java
import java.util.Scanner;

public class OddOrEven {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter a Number :-");
        int a = scanner.nextInt();
        if (a % 2 == 0) {
            System.out.println(a + " is Even.");
        } else {
            System.out.println(a + " is Odd.");
        }

        scanner.close();
    }
}

File: client/public/Codes/C/OddOrEven.c
#include <stdio.h>

void CheckOddEven(int num)
{
    if (num % 2 == 0)
    {
        printf("%d is Even.", num);
    }
    else
    {
        printf("%d is Odd.", num);
    }
}

int main()
{
    int num;
    printf("Enter a Number:");
    scanf("%d", &num);

    CheckOddEven(num);
    return 0;
}
